kernel/sysfile.c

```diff
@@ -16,6 +16,8 @@
 #include "file.h"
 #include "fcntl.h"
 
+_Atomic uint64 readcount;
```

Comment:
```suggestion
static _Atomic uint64 readcount;
```

피드백: `static` modifier가 있으면 좋겠다.

Reply:
https://github.com/EcmaXp/xv6-riscv/pull/1/commits/8c44c06da77fb2c16f11d6418e99810e493bf73a